frontend/src/utils.ts

```diff
@@ -36,6 +36,7 @@ export type VmStatus =
   | 'CreationLoopBackoff'; //the environment has encountered a temporary error during creation.
 export type Instance = {
   id: number;
+  idName: string;
```

Comment:
I think it could be cleaner to make a prettyName property, so that name reflects the metadata.name from the original instance

Reply:
I think could be a good idea but in a next pr because now @cannarelladev is working using those properties and i'm not the one that managed the instance rows.

Reply:
I'd prefer avoid adding parameters now and then change/remove them with another PR, we could try to see if @cannarelladev can sync the change already in the #614

Reply:
Ok

Reply:
I talked with @cannarelladev and we think that could be a better idea to add the part relative to the pretty name in a next PR as decided before.